woodwork/utils.py

```diff
@@ -216,6 +216,10 @@ def _is_url(string):
 
 def _reformat_to_latlong(latlong, use_list=False):
     """Reformats LatLong columns to be tuples of floats. Uses (np.nan, np.nan) for null values."""
```

Comment:
The docstring for this needs to be updated now that we can also have a single `nan` value. Probably should also review all the other latlong related docstrings to make sure they are consistent with the new behavior as well.